scripts/sync_world_map_langs.py

```
@@ -0,0 +1,522 @@
+            mid = entry.msgid.rstrip("\n")
+            if mid not in official:
+                continue
+            new = official[mid].get(lang_key)
+            if not new:
+                continue
+            if entry.msgstr != new:
+                touched.append((mid, entry.msgstr, new))
+                entry.msgstr = new
+                stats += 1
+
+        # 3) 内嵌替换（按 zh 名长度降序，避免子串冲突）
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)